test/metabase/server/middleware/offset_paging_test.clj

```diff
@@ -48,8 +48,37 @@
             (read-response (handler (ring.mock/request :get "/" {:offset "200", :limit "100", :whatever "true"}))))))
   (testing "w/ non-numeric paging params, paging is disabled"
     (is (=? {:status 200
-             :body {"limit" nil
+             :body {"limit"  nil
                     "offset" nil
                     "paged?" false
                     "params" {}}}
             (read-response (handler (ring.mock/request :get "/" {:offset "foo" :limit "bar"})))))))
+
+(deftest page-result-test
+  (let [numbers (range 100)]
+
+    (testing "no paging params"
+      (is (= numbers (page-result numbers))))
+
+    (testing "offset but no limit"
+      (binding [mw.offset-paging/*offset* 10]
+        (is (= numbers (page-result numbers)))))
```

Comment:
I find this surprising, it'd expect the absence of an offset to be treated as just a 0 offset.